### PR TITLE
Migrate from "import" to "use", avoid division in stylesheets and switch from sass-lint to stylelint

### PR DESCRIPTION
**Pull request by maciaszczykm**
Closes https://github.com/kubernetes/dashboard/issues/6089.
Closes https://github.com/kubernetes/dashboard/issues/6204.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -18,5 +18,6 @@ require (
 	k8s.io/apiextensions-apiserver v0.21.0
 	k8s.io/apimachinery v0.21.0
 	k8s.io/client-go v0.21.0
+	k8s.io/code-generator v0.21.0 // indirect
```
Comment (maciaszczykm) [inline go.mod:21]: Auto generated by `npm ci`.

File: package.json [hunk 2]
```diff
@@ -189,7 +189,7 @@
   },
   "engines": {
     "node": ">=14.16.1",
-    "npm": ">=6.0.0 <7.0.0"
+    "npm": ">=6.0.0"
```
Comment (maciaszczykm) [inline package.json:192]:
```➜ npm --version
7.7.6

 ➜ node --version              
v15.14.0
```

Works for me.

Comment (shu-mutou) [reply package.json:192]: F.Y.I. Same errors of static check occur in env using `./aio/develop/run-npm-on-container.sh`. In this env, versions are as follows:
node: 14.16.1
npm: 6.14.12

But e2e test and dashboard works properly.

Comment (shu-mutou) [reply package.json:192]: I could not find the way to fix it today. Have a nice weekend.

Comment (maciaszczykm) [reply package.json:192]: You too, I'll try to investigate next week too.

Comment (maciaszczykm) [reply package.json:192]: I've reverted it for now. Let's focus on LTS only.

File: aio/gulp/backend.js [hunk 3]
```diff
@@ -30,8 +30,6 @@ gulp.task('backend', gulp.series((doneFn) => {
   goCommand(
       [
         'build',
-        // Install dependencies to speed up subsequent compilations.
-        '-i',
```
Comment (floreks) [inline aio/gulp/backend.js:34]: Any reason to remove this?

**Comment by shu-mutou** [comment]
LGTM except for floreks questions. thanks!

File: aio/gulp/backend.js (hunk 3 above)
Comment (maciaszczykm) [reply aio/gulp/backend.js:34]: Yes, there is warning in the build logs that this flag is deprecated. From https://golang.org/cmd/go/: "The -i flag installs the packages that are dependencies of the target. The -i flag is deprecated. Compiled packages are cached automatically.".

**Pull request merged**